addons/hr_org_chart/views/hr_views.xml

```diff
@@ -6,9 +6,17 @@
         <field name="inherit_id" ref="hr.view_employee_form"/>
         <field name="arch" type="xml">
             <div id="o_work_employee_main" position="after">
-                <div id="o_employee_right">
-                    <h4 class="o_org_chart_title mb16 mt0">Organization Chart</h4>
-                    <field name="child_ids" widget="hr_org_chart" readonly="1"/>
+                <div id="o_employee_right" class="col-lg-4 px-0 ps-lg-5 pe-lg-0">
+                    <table class="o_group o_inner_group">
+                        <tbody>
+                            <tr>
+                                <td class="pe-lg-0">
+                                    <div class="o_horizontal_separator o_org_chart_title">Organization Chart</div>
+                                    <field name="child_ids" class="position-relative" widget="hr_org_chart" readonly="1"/>
+                                </td>
+                            </tr>
+                        </tbody>
+                    </table>
```

Comment:
What do you think about:
```suggestion
                    
                    <group>
                        <div class="o_horizontal_separator o_org_chart_title" colspan="2">Organization Chart</div>
                        <field name="child_ids" class="position-relative" widget="hr_org_chart" readonly="1" nolabel="1"/>
                    </group>
```

Reply:
Confirmed, much better, thanks.
Implemented

Cc. @adr-odoo